community-membership.md

```diff
@@ -220,6 +219,26 @@ The following apply to the subproject for which one would be an owner.
 - Work with other maintainers to maintain the project's overall health and
   success holistically.
 
+### Becoming a Maintainer
+
+Unless stated otherwise in a SIG charter ratified by the Technical Committee,
+a new maintainer is elected by unanimous vote of the existing maintainers
```

Comment:
I don't have a strong opinion about the "unanimous vote" part, but for the reference the Jaeger's process is a bit more relaxed:

> Two other maintainers need to second the nomination. If no one objects in 5 working days (U.S.), the nomination is accepted. If anyone objects or wants more information, the maintainers discuss and usually come to a consensus (within the 5 working days). If issues can't be resolved, there's a simple majority vote among current maintainers.

Reply:
Our project is broken into many small groups who each maintain only one portion with only a few maintainers. Unanimous consent of one SIG represents only a small portion of the overall maintainers and shouldn't be too high of a bar I think.

Reply:
I like the lazy consensus part of @yurishkuro 's quote. Active consensus sometimes hard to get.

Reply:
For most things I'd agree but it seems like adding a maintainer is about the most impactful thing you can do. If a maintainer can't be bothered to consent to a new maintainer either the new maintainer isn't agreed on or the existing maintainer isn't doing their job imo

Reply:
> or the existing maintainer isn't doing their job

This might indeed happen: maintainers who have moved on might not have yet turned themselves into emeritus and might be hard to reach. Same for maintainers on longer breaks.

Reply:
So how about if there is a unanimous vote then the vote passes, otherwise if there is no objection within 5 days the vote passes?

Reply:
I updated the PR to reflect this lazy consensus. PTAL